scipy/stats/tests/test_distributions.py

```diff
@@ -3450,6 +3450,18 @@ def test_burr12_ppf_small_arg():
     #   5.7735026918962575e-09
     assert_allclose(quantile, 5.7735026918962575e-09)
 
+def test_burr_nan_mean_var():
+    c, d = 0.5, 3
+    mean, variance = stats.burr(c, d).stats()
+    assert_(np.isnan(mean))
+    assert_(np.isnan(variance))
+
+def test_burr_stats():
+    c, d = 5.0, 3
+    mean, variance = stats.burr(c, d).stats()
+    mean_hc, variance_hc = 1.4110263183925857, 0.22879948026191643
+    assert_allclose(mean_hc, mean_hc)
```

Comment:
Typo,  `mean, mean_hc`

Reply:
Yes, o/w this is the most useless unit test ever :)